client/src/css/ResourcePreview.css

```diff
@@ -1,22 +1,28 @@
-.resource-preview-border:hover {
-  box-shadow: 3px 3px 5px #b6b6b6;
-}
-
-.resource-preview-cover-img {
-  width: 100%;
-  height: 100%;
-  display: flex;
-  justify-content: center;
-}
-
-.resource-preview-text:hover {
-  text-decoration: none;
-}
-
-.resource-preview-name {
-  display: inline-block;
-  max-width: 80%;
-  text-overflow: ellipsis;
-  overflow: hidden;
-  vertical-align: middle;
-}
+.resource-preview-border:hover {
+  box-shadow: 3px 3px 5px #b6b6b6;
+}
+
+.resource-preview-cover-img {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.resource-preview-text:hover {
+  text-decoration: none;
+}
+
+.resource-preview-name {
+  display: inline-block;
+  max-width: 80%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  vertical-align: middle;
+}
+
+.description {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
```

Comment:
nit - can this be named something like `resource-preview-description`? I believe css can get weird with it kind of being global, so it's best to be descriptive with what component the CSS is relevant for